ftml/src/render/html/element/iframe.rs

```diff
@@ -33,7 +33,11 @@ pub fn render_iframe(
         "url" => url,
     );
 
-    ctx.html().iframe().attr("src", &[url]).attr_map(attributes);
+    ctx.html()
+        .iframe()
+        .attr("src", &[url])
+        .attr("crossorigin", &[])
```

Comment:
Out of curiosity, does Rust not have default parameter values? Or does it handle that with overloading? In JS I would make the second argument optional so you could just do `attr("crossorigin")`. Not a suggestion, just curious.

Reply:
No, it does not. Rust is very big on making things explicit, and default parameter values aren't very apparent when looking at usages of a function or method. If you want to have a default value then you create a new method with a clear name. For instance, `Vec::new()` vs `Vec::with_capacity()`.

Similarly, it does not have overloading. (Honestly I think overloading as a general concept is a mistake, like in C++ it requires name mangling and type inferences to work, which massively overcomplicates things for relatively little gain).

I suppose I could add `attr_null()` or something for attributes which only have keys, but honestly I think this is clear enough.

Reply:
That makes sense. I suspect that `&[]` would get compiled out anyways.

Fun fact: Typescript _adds overloading_. JS doesn't have it at all, and the way TS overloading works is purely on the type level. What's funny is that I generally don't like it there too. Usually it just adds some confusion to the function signature because you don't immediately know how you're supposed to input arguments.

If JS was redesigned, it would probably have named arguments by default.